rllib/core/testing/torch/bc_optimizer.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, List, Mapping
+
+import torch
+
+from ray.rllib.core.optim.rl_optimizer import RLOptimizer
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.nested_dict import NestedDict
+
+
+class BCTorchOptimizer(RLOptimizer):
+    def __init__(self, module, config):
+        super().__init__(module, config)
+
+    @override(RLOptimizer)
+    def compute_loss(
+        self, batch: NestedDict, fwd_out: Mapping[str, Any]
+    ) -> Mapping[str, Any]:
+        """Compute a loss"""
+        action_dist = fwd_out["action_dist"]
+        actions = batch["actions"]
+        return {"total_loss": -action_dist.log_prob(actions.view(-1)).mean()}
```

Comment:
Can you return a single tensor here for more clarity?

From `RLOptimizer.compute_loss()`.
```
            Either a single loss tensor which can be used for computing
            gradients through, or a dictionary of losses. NOTE the dictionary
            must contain one protected key "total_loss" which will be used for
            computing gradients through.
```

Reply:
yeah it was not truly supported as it depends on the implementation of RLTrainer. I have made it like you suggested in the unittest, but this is a TODO for @avnishn When designing the RLTrainer.